android/src/main/java/com/magicleap/magicscript/scene/nodes/dropdown/UiDropdownListNode.kt

```diff
@@ -151,25 +160,37 @@ class UiDropdownListNode(
         }
     }
 
+    private fun setMultiSelect(props: Bundle) {
+        val multiSelect = props.read<Boolean>(PROP_MULTI_SELECT)
+        if (multiSelect != null) {
```

Comment:
```
val multiSelect = props.read<Boolean>(PROP_MULTI_SELECT) ?: false
listNode.multiSelect = multiSelect
```
looks better, what do you think?

Reply:
Looks better but will not work correctly: what if it's set to true and then user updates some properties (e.g. maxCharacters)? Then PROP_MULTI_SELECT is absent in `props` and `multiSelect` incorrectly will be set to false.

Reply:
good catch! you're right